docs/MVS_Outputs.rst

```diff
@@ -170,6 +211,6 @@ MVS version number, the branch ID and the simulation date are provided as well i
 A commit hash number is provided at the end of the report in order to prevent the erraneous comparing results from simulations using different versions.
 
 It includes several tables with project data, simulation settings, the various demands supplied by the user, the various components of the system and the optimization results such as the energy flows and the costs.
-The report also provides several plots which help to visualize the flows and costs. The PDF report can be generated by running the command (details in the READTHEDOCS `here <https://github.com/rl-institut/multi-vector-simulator/blob/dev/README.md#generate-report>`_)::
+The report also provides several plots which help to visualize the flows and costs. The PDF report can be generated by running the command (details in the READTHEDOCS `here <https://github.com/rl-institut/mvs_eland/blob/dev/README.md#generate-report>`_)::
```

Comment:
It should be multi-vector-simulator here now - https://multi-vector-simulator.readthedocs.io/en/latest/?badge=latest